src/components/Tabs/index.js

```diff
@@ -22,7 +22,7 @@ export default class extends Component {
     return (
       <Tabs onSelect={this.handleSelect}>
 
-        <TabList className='list pa0 ma0 bg-light-navy flex-ns br2 br--top'>
+        <TabList className='list pa0 ma0 bg-light-navy flex-ns br2 br--top bb b--navy'>
```

Comment:
Adds separator line between tab headers and panes:
![image](https://cloud.githubusercontent.com/assets/5497885/23436737/e4707338-fdc8-11e6-842b-7713e3d03e21.png)